paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/customersheet/playground/CustomerSheetPlaygroundViewState.kt

```diff
@@ -0,0 +1,25 @@
+package com.stripe.android.paymentsheet.example.samples.ui.customersheet.playground
+
+import com.stripe.android.customersheet.ExperimentalCustomerSheetApi
+import com.stripe.android.customersheet.PaymentOptionSelection
+
+@OptIn(ExperimentalCustomerSheetApi::class)
+sealed class CustomerSheetPlaygroundViewState(
+    open val isSetupIntentEnabled: Boolean = true,
+    open val isGooglePayEnabled: Boolean = true,
+    open val isExistingCustomer: Boolean = true,
```

Comment:
These don’t really make sense outside of `Data`. Can you move them?

Reply:
It is for convenience, I will leave this for now.